packages/ozone/src/api/set/deleteValues.ts

```diff
@@ -0,0 +1,31 @@
+import { AuthRequiredError, InvalidRequestError } from '@atproto/xrpc-server'
+import { Server } from '../../lexicon'
+import AppContext from '../../context'
+
+export default function (server: Server, ctx: AppContext) {
+  server.tools.ozone.set.deleteValues({
+    auth: ctx.authVerifier.modOrAdminToken,
+    handler: async ({ input, auth }) => {
+      const access = auth.credentials
+      const db = ctx.db
+      const { name, values } = input.body
+
+      if (!access.isAdmin) {
+        throw new AuthRequiredError(
+          'Must be a moderator to remove values from a set',
+        )
+      }
```

Comment:
Perhaps this is intended to be `access.isModerator`— would match perms on other set endpoints and this error message.
```suggestion
      if (!access.isModerator) {
        throw new AuthRequiredError(
          'Must be a moderator to remove values from a set',
        )
      }
```